core/src/main/java/org/opensearch/sql/expression/function/PPLFuncImpTable.java

```diff
@@ -531,6 +531,11 @@ void populate() {
                       builder.makeLiteral(" "),
                       arg),
               PPLTypeChecker.family(SqlTypeFamily.STRING)));
+      register(
+          ATAN,
```

Comment:
Could you elaborate why we register ATAN2 as ATAN? I see both Calcite and SparkSQL has ATAN and ATAN2 separately: https://calcite.apache.org/docs/reference.html, https://spark.apache.org/docs/latest/api/sql/#atan

Reply:
I see. Sure, we can go ahead if we follow aligning with V2 criteria for all functions. Just feel this may cause unnecessary work in both Calcite and Spark. We can discuss later. Thanks!